Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +11,80 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            if (string.IsNullOrEmpty(message))
+            {
+                return message;
+            }
+
+            return TransformMessage(message, shift);
         }
 
         public static string Decrypt(string message, byte shift)
         {
-            return message;
+            if (string.IsNullOrEmpty(message))
+            {
+                return message;
+            }
+
+            return TransformMessage(message, shift * -1);
+        }
+
+        private static string TransformMessage(string message, int shift)
+        {
+            var cipheredMessage = new StringBuilder();
+
+            for (var i = 0; i < message.Length; i++)
+            {
+                cipheredMessage.Append(GetNewLetter(message[i], shift));
+            }
+
+            return cipheredMessage.ToString();
+        }
+
+        private static char GetNewLetter(char letter, int shift)
+        {
+            if (!IsLetter(letter))
+            {
+                return letter;
+            }
+
+            var letterPosition = GetLetterPosition(letter);
+
+            var newLetterPosition = (letterPosition + shift) % 26;
```

Comment:
26 is a magic number. What if I asked you to increase the amount of all possible characters? You would need to change multiple places. Please use a constant instead.